TODO/the-future-of-ux-design.md

```diff
@@ -1,57 +1,56 @@
+> 隐藏在解决病人病情背后的设计理念正在进行着由被动向主动的改变。
 
-> The design concept behind it being more proactive rather than reactive, in terms of addressing a patient’s condition.
+比如，医生将可以直接建议你多开窗以改变你办公环境的空气质量，或者是提醒你在工作的同时别忘了加一些小间隔的休息，而不是给你开一些处方药了事。你休息或工作时所处的环境和你的身体症状一样都会被一声考虑在内。
 
-Rather than prescribing pills, a doctor could suggest a change in the air quality of your office or more frequent breaks given the nature of your work. The environment where you rest or exercise will be taken into consideration as well as the physical symptoms of your condition.
+医院可以在你的工作地点进行一些常规检查，并且可以将医疗中心部署在人员密集的地区。健康的生活习惯将会被鼓励，而不健康的将会在某些地方有“小小的惩罚”来督促你改变不良的习惯。
 
-Hospitals will implement check up routines in your workspace, and put health care centers closer to places of mass congregation. Good healthcare habits will be rewarded, and negative ones taxed in some way to motivate a change in habits.
+### 3. 可持续性
 
-### 3. Sustainability
+计划性报废的产品将不复存在于这个资源短缺的世界上了。产品必须经久耐用，甚至要做到比它的设计者有着更长的寿命。
 
-Planned obsolescence will no longer be an option in a world with scarce and limited resources. Products must be long lived, timeliness and functional beyond the life span of their designer.
+未来的设计师们必须要考虑可持续性，并且保持对生产，材料和所用资源负责任。
 
-Designers of the future must care about sustainability and maintain responsibility for the processes, materials and sourcing used.
+### 4. 更聪明的机器
 
-### 4. Smarter Machines
+自动化和机器学习将会在你的用户体验中占据大半江山。它们可能会在一些常规进程或活动中预测感知到下一步该如何做，并且提醒你。也可能通过学习一些诸如“你每三周都要做的事”来取代你完成这些微不足道的日常琐事。
 
-Automation and machine learning will play a larger role in your experience with a product. Whether its sensing what tasks are the next steps in a common progression and thus prompting you to do them, it may simply take over the most mundane tasks that are a part of your routine, simply through “learning” that the tasks occur every third week or so.
+> 我们丰富的用户数据能使你的产品们能相互交流，来为你提供无缝的用户体验。
 
-> Our wealth of user data plays a major role in the sense that your products can “speak” to each other to deliver a seamless experience.
+> 举个例子，你手机上的一个应用可以追踪你的健康信息并提醒你的医生。你的医生会直接将一些医疗指导或建议发到你妻子的邮箱里。
 
-> For example, the app on your mobile device can track your health data and relay it to your doctor who inputs some instructions that go directly to an inbox for your wife.
+自然语言可以被更多机器理解。你再也不用有必须将大段内容浓缩至简后才能搜索的挫败感了，取而代之的是针对对话内容的搜索。
 
-Natural language will be understood by more machines and the frustrations related to searching for something that you can succinctly condense into a few lines of text for a search engine, will be replaced by a dialogue.
+随着机器越发的人性化，机器将会可以预测我们的喜怒哀乐、需求和爱好。而我们可能会越发的“机械化”。
 
-As machines become more human, in the sense that they can predict our emotional ups and downs as well as our needs and habits, we as humans become more similar to a machine.
+> 未来城市的生活体验将会更加温和和具有流动性，正如我们的工作生态系统。
 
-> Experiences in the cities of the future will flow gracefully and fluidly, as part of a working ecosystem.
+便利性将会全面碾压传统的去一个地方只能办一两件事的状况。正如医疗服务一样，多面性的建筑设计会解决大量的人流，运输多种不同的货物，提供不同的服务和娱乐。
```

Comment:
> Instead of moving from one physical location to another location for different needs, convenience will be king.

便利性将会成为主导, 而非为了不同的需求往返于不同的地点. 

>  Multi-faceted building designs will serve the masses, delivering a variety of different goods, products, services and entertainment, as well as healthcare services.

多方面的大楼设计将为广大群众服务, ...... 也有医疗保健服务. 

![image](https://cloud.githubusercontent.com/assets/1016943/17779884/dc5b60ec-659b-11e6-8aec-575019f67a33.png)